.flake8

```diff
@@ -53,6 +53,8 @@ per-file-ignores =
   # If other ignores are added for a specific file in the section following this,
   # these will need to be added to that line as well.
 
+  # Q000: Allow single-quotes in _version.py because setuptools_scm will create it with them.
```

Comment:
(educational) FTR that template *could* be changed with our own. Or maybe even contributed to the default template upstream. And even include comment-based typing information.
Of course, this is probably not something to include in this PR. JFYI. Feel free to "resolve" once you see this.